examples/pyspark_ml_autologging/one_vs_rest.py

```diff
@@ -0,0 +1,23 @@
+from pyspark.ml.classification import LogisticRegression, OneVsRest
+from pyspark.ml.linalg import Vectors
+from pyspark.sql import SparkSession
+from sklearn.datasets import load_iris
+
+import mlflow
+
+spark = SparkSession.builder.getOrCreate()
+X, y = load_iris(return_X_y=True)
+df = spark.createDataFrame(
+    [(Vectors.dense(features), int(label)) for features, label in zip(X, y)], ["features", "label"]
```

Comment:
```suggestion
    [(Vectors.dense(features), label) for features, label in zip(X, y)], ["features", "label"]
```

Withtout `int`, we'd get this error because `label` is of `numpy.int64`:

```
TypeError: not supported type: <class 'numpy.int64'>
```

Reply:
This loading data style in pyspark ML is not recommended. it has low performance and consume much memory

We recommend load data by "libsvm" format from file, like:
```
spark.read.format('libsvm').load('file:/xxxx/...')
```
sample file:
https://github.com/apache/spark/blob/master/data/mllib/iris_libsvm.txt

another approach is:
convert the loaded iris data into pandas dataframe, then create spark dataframe by:
```
spark.createDataFrame(pandas_dataframe)
```

Reply:
Offline discussion: we decided to use `spark.createDataFrame(pandas_dataframe)` + `VectorAssmbler` .